sw/device/lib/dif/dif_clkmgr.h

```diff
@@ -148,6 +148,20 @@ dif_result_t dif_clkmgr_hintable_clock_get_hint(
     const dif_clkmgr_t *clkmgr, dif_clkmgr_hintable_clock_t clock,
     dif_toggle_t *state);
 
+/**
+ * Enable chip to use the external clock.
+ *
+ * The input arg is_low_speed indicates what speed the external clocks is.
+ * High speed - external clock is close to nominal speeds (e.g. external clock
+ * is 96MHz and nominal frequency is 96MHz-100MHz). Low speed - external clock
+ * is half of nominal speeds (e.g. external clock is 48MHz and nominal frequency
+ * is 96MHz-100MHz).
+ *
+ */
+OT_WARN_UNUSED_RESULT
+dif_result_t dif_clkmgr_enable_external_clock(const dif_clkmgr_t *clkmgr,
```

Comment:
nit: may want to stay consistent with the naming of other `*_get/set_enabled` DIFs for this IP, i.e., rename this `dif_clkmgr_external_clock_set_enabled(...)`.

Reply:
Fixed. Thanks